src/System.Data.SqlClient/src/Interop/LocaleMapperApiHelper.cs

```diff
@@ -0,0 +1,482 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Also, these files should probably be named as LocaleApiHelper.Windows.cs and LocaleApiHelper.Unix.cs to fit our naming conventions.